extensions/default/src/getHangingProtocolModule.js

```diff
@@ -65,7 +72,11 @@ const defaultProtocol = {
           viewportOptions: {
             viewportType: 'stack',
             toolGroupId: 'default',
-            // initialImageOptions: {
+            initialImageOptions: {
+              type: 'custom',
+              attribute: 'sopInstanceLocation',
+            },
```

Comment:
first of all we shouldn't change this since it is the default HP, I think just the documentation is enough. Both in PR and the MD docs

Reply:
can we mix this together? so we have 

``` js
initialImageOptions = {
  index: 180
}
```

or 


```js
initialImageOptions = {
  preset: 'middle'
}
```

or

```js
initialImageOptions = {
  custom: 'sopInstanceLocation'
}
```